packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/barriers-table.jsx

```diff
@@ -0,0 +1,101 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import { CSSTransition } from 'react-transition-group';
+import { Icon, Text, ThemedScrollbars } from '@deriv/components';
+import Fieldset from 'App/Components/Form/fieldset.jsx';
+import BarriersList from './barriers-list.jsx';
+
+const BarriersTable = ({
+    barrier_1,
+    onChange,
+    setHoveredBarrier,
+    is_barriers_table_expanded,
+    turbos_barrier_choices,
+    toggleBarriersTable,
+}) => {
+    const [chosen_barrier, setChosenBarrier] = React.useState(barrier_1);
+
+    const changeBarrier = e => {
+        if (e.target.id) {
+            e.stopPropagation();
+            setHoveredBarrier(null);
+            setChosenBarrier(e.target.id);
+            onChange({
+                target: {
+                    name: 'barrier_1',
+                    value: e.target.id,
+                },
+            });
+        }
+    };
+
+    const onMouseOver = e => {
```

Comment:
I don't like these 2 functions (`onMouseOver`, `onMouseOut`), but there is no event for hover in JS. The `onMouseOver` event occurs when the mouse pointer enters an element and  `onMouseOut` occurs when the mouse pointer leaves the element. The `onMouseOver` event is similar to the `onMouseEnter` event. The difference is that the `onMouseEnter` event doesn't bubble and **do not call if mouse move from parent to child**.

Reply:
@kate-deriv please use onMouseEnter and onMouseLeave same like we do in some other components (please check PositionsDrawerCard for example), these events will fire less often which is much better for performance )

Reply:
`onMouseEnter` won't be called, if we put it on `<ul>` and hover between `<li>`. In that case we need to apply `onMouseEnter` to every `<li>` .
Events in JavaScript provide a dynamic interface to the webpage. There are wide variety of events such as user clicking, moving the mouse over an element, etc. Events that occur when the mouse interacts with the HTML document falls under the category of MouseEvent property.
[Explanation link](https://www.geeksforgeeks.org/difference-between-mouseover-mouseenter-and-mousemove-events-in-javascript/)
Example from that link:
**mouseover**: The onmouseover event triggers when the mouse pointer enters an element or any one of its child elements.
**mouseenter**: The onmouseenter event is triggered only when the mouse pointer hits the element.

Reply:
So I'll apply `onMouseEnter` and `onMouseLeave` to each `<li>` + remove `e.stopPropagation();` as those 2 event do not propagate.